code/src/pages/LandingPage.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { useNavigate } from "react-router";
+import styled from "styled-components";
+
+import { game } from "../reducers/game";
+import { CompassAnimation } from "../components/CompassAnimation";
+import Button from "../components/Button";
+
+const TextContainer = styled.form `
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-size: 30px;
+
+    & input {
+      font-size: inherit;
+      font-family: inherit;
+      border-radius: 2px;
+      border: none;
+    }
+`
+
+const LandingPage = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const [input, setInput] = useState("");
+
+  const onButtonClick = (username) => {
+    dispatch(game.actions.setUserName(username));
+    navigate("/start");
```

Comment:
This is very useful hook, instead of using Link in the return, you use navigate hook, is it so? Looks like you like router as well! :)

Reply:
useNavigate is the updated version of useHistory. https://dev.to/salehmubashar/usenavigate-tutorial-react-js-aop